README.md

```diff
@@ -60,14 +60,16 @@ Usage: lint-staged [options]
 
 Options:
   -V, --version                      output the version number
-  --allow-empty                      allow empty commits when tasks undo all staged changes (default: false)
+  --allow-empty                      allow empty commits when tasks revert all staged changes (default: false)
   -c, --config [path]                path to configuration file
   -d, --debug                        print additional debug information (default: false)
+  --no-backup                        disable the backup stash, and do not revert in case of errors
```

Comment:
Much better name!

Reply:
This caused a bit of an oops because commander inverts `--no-backup` to `!cmdline.backup` and left `cmdline.noBackup === undefined`.